app/Http/Controllers/AlbumController.php

```diff
@@ -384,14 +392,25 @@ function merge(Request $request)
 		$no_error = true;
 		foreach ($photos as $photo) {
 			$photo->album_id = $albumID;
+			if ($this->sessionFunctions->is_admin()) {
```

Comment:
actually this check is unnecessary. If you are a normal user:
`$photo->owner_id` = `$album->owner_id`
So the result will be the same.